### ISSUE TITLE
CubicSDR doesn't see the device

### ISSUE THREAD

**Issue by serj053** [cjcliffe/CubicSDR#787]
Does not detect devices. Although the installation was performed exactly according to the site's instructions. To add modules, I used the method of reusing modules and libraries from an existing installer. But the most unpleasant thing is that even the application itself installed by the installer gives the same error. Help to understand and launch the app.
![not_found_divice](https://user-images.githubusercontent.com/10112259/72785915-5847e600-3c3d-11ea-8f8b-eda7de76ccb6.png)

**Comment by vsonnier** [comment]
You may have need to install additional libraries where Cubic can find them. But there is an easier way because you use Windows: you can install the [PothosSDR distribution](https://github.com/pothosware/PothosSDR/wiki/Tutorial) (Resources / Download installer) which contains a Win64 build of the latest CubicSDR source code together with all the existing SoapySDR modules to talk to any SDR device, together with the up-to-date required libraries.

**Comment by vsonnier** [comment]
@serj053 Hi ! Did you try the PothosSDR distribution, and does it work for you ? Also, please don't open multiple issues on the same theme like #790, and continue here.

**Comment by serj053** [comment]
Hello. Yes, of course, I will take into account your wishes. The Photos app works great, but I'm interested in figuring out how Cubic is built and how its elements work. That's right : https://github.com/cjcliffe/CubicSDR/wiki/Build-Windows but I collected Cubic, collected already four times and always on this place "LibUSB 1.0 required to compile rtlsdr" I stumble, maybe there are some subtleties, although it may not be my care, tomorrow I will collect everything from the beginning.

**Comment by serj053** [comment]
I reassembled the Cubic again and when I reached the "libusb" item, I executed both of the proposed options, but still , but still in both cases, CMake throws an error "CMake Error at CMakeLists.txt:77 (message):  LibUSB 1.0 required to compile rtl-sdr" when executing the "librtlsdr" item. Why it might be.

**Comment by vsonnier** [comment]
Copy of your previous message:

![Error_librtlsdr_CMake](https://user-images.githubusercontent.com/10112259/73550366-83a8ad00-4455-11ea-80cc-b4f699b319a8.png)

> Friends, please tell me what this error message means.

You did not filled all the paths required for `librtlsdr`: https://github.com/cjcliffe/CubicSDR/wiki/Build-Windows#librtlsdr, here visibly the path to THREADS_PTHREADS_INCLUDE_DIR was not filled. 

Just RTFM starting from here: https://github.com/cjcliffe/CubicSDR/wiki/Build-Windows#pthreads-win32 and below, and don't skip steps you will get there eventually.

**Comment by serj053** [comment]
Thank you for your help, I am so risking soon to master the English language perfectly)). But unfortunately I have no success. I tried to show how I completed the previous points
![pthreads_issuse](https://user-images.githubusercontent.com/10112259/73596547-ec665700-4533-11ea-8f81-9463f8c580f1.png)

![libusb](https://user-images.githubusercontent.com/10112259/73596561-06a03500-4534-11ea-94fc-fdbe3c3284f5.png)
Following your comment didn't fix the situation. When I run CMake, the following lines should appear in the interface according to the description map - " LIBUSB_INCLUDE_DIRS ", "THREADS_PTHREADS_WIN32_LIBRARY" and "LIBUSB_LIBRARIES", but they do not appear. When I enter these lines through the "ADD" operation (STRING + FILEPATH), they disappear after configuration. I guess I should go to sleep and try again while no one is around.
![step_1_CMake](https://user-images.githubusercontent.com/10112259/73596782-92b35c00-4536-11ea-9cd1-c5ca90e35dc8.png)

![step_2_CMake](https://user-images.githubusercontent.com/10112259/73596785-9810a680-4536-11ea-9798-6431ab65982f.png)
